monero-outreach-docs/translations/es/merchants/merchant_faq_es.md

```diff
@@ -0,0 +1,58 @@
+
+### _¿Qué diferencia hay entre aceptar Monero a aceptar Bitcoin?_
+
+La configuración tanto de Bitcoin como de Monero son similares, pero las cadenas de bloques son diferentes. Monero tiene algunas características únicas que le podrían interesar más a los comerciantes que son:
+
+- **Pagos más rápidos** - Monero tiene un tiempo de verificación de 2 minutos por bloque a diferencia de los 10 minutos por bloque en Bitcoin. En promedio, tu transacción llegaría 5 veces más rápido a la cadena de bloques de Monero que a la de Bitcoin.
+- **Menos costoso** - Monero tiene comisiones mucho más bajas que Bitcoin y además, Monero continúa siendo cada vez más eficiente, la tendencia en términos de comisiones es tener comisiones más bajas que las de Bitcoin.
+- **No hay retrasos en las transacciones** - Los bloques de Monero son dinámicos mientras que Bitcoin tiene bloques fijos. Cuando la red de Bitcoin está ocupada y los bloques están llenos, las transacciones deben, o pagar comisiones altas, o retrasarse.
+- **Libre de riesgo** - Siendo que Monero es privado por defecto y por ende fungible, nadie puede discriminar el historial de las monedas. Las monedas marcadas como contaminadas en la red de Bitcoin es un problema creciente para los comerciantes, pero Monero no tiene ese problema.
+
+### _¿Cómo se comparan las comisiones de Monero con las comisiones de las tarjetas de crédito?_
+
+Las comisiones en la red de Monero son pagadas por quien envía la transacción. Recibir XMR siempre será gratis. Al cliente le cuesta aproximadamente $0.02 centavos de dólar pagar con Monero. Pagar con Monero es menos costoso que una transacción con tarjeta de crédito, además de tener un procesamiento mucho más rápido.
+
+| Comisión | American Express | Mastercard | Visa | Débito |
+|--|--|--|--|--|
+| Comisión de Procesamiento | 2.5% - 3.5% | 1.55% - 2.6% | 1.43% - 2.4% | 0.30% - 2.75% |
+| Comisión de intercambio | -- | 0.08% | 0.09% | 0.05% + $0.21 0.70% + $0.15 |
+| Comisión de la red | 0.15% | 0.08% | 0.09% | 0.75% - 0.90% |
+| Comisión Switch | -- | -- | -- | $0.025 - $0.075 |
+| Comisiones por compras de $10 USD | **$0.27 - $0.37** | **$0.18 - $0.28** | **$0.16 - $0.26** | **$0.35 - $0.66** |
```

Comment:
I'd remove the USD symbol. It's duplicate with $

Reply:
Well, the main reason I left it is because it's a reference to the USD, in other countries the symbol **'$'** represents money but it doesn't make a reference to a currency. E.g, the 'peso' doesn't have a symbol like the EUR (€) or the GBP (£), and its monetary representation comes accompanied with the **'$'** sign.

Reply:
So, removing the '$' symbol could confuse readers' conversion rate.

Reply:
I know, but you're using the symbol two times: $ and USD. That's why I would keep with one of these.

Reply:
Okay, I understand. I'll remove it.

Reply:
You're right, I'll removed the '$' symbol.